### ISSUE TITLE
License Section

### ISSUE THREAD

**Issue by RichardLitt** [RichardLitt/standard-readme#14]
@tunnckoCore, [here](https://github.com/feross/standard/issues/141#issuecomment-221971973):

> And second I also don't have License section, because it 1) can be accessed directly by opening the LICENSE file from file browser, or 2) by clicking the badge which is next to the name of the repo (h1). I think it is more faster than scrolling to the end of big/small readmes. Look at mukla's or hybridify's current/latest readme using verb 0.9. :)

> If you are lawyer or such, or just you come to see the license, isn't it more natural to look is there a license file first and open it directly, instead of scrolling and etc? That's why I don't have license section. Also in the package's npm page have link to license based on license field in package.json.

I still think it is industry standard - at least in the Node community - to have a License note, and as it is the last element, I don't think it detracts. The README should be the sole source of information - it can link elsewhere (like to the LICENSE file), and I think this redundancy (which you are right, it is) is useful, in that regard. If I just `cat` the README, I should know the License without having to look in another file. Also, package.json is js-specific. I want standard-readme to be more than just javascript.

**Comment by wooorm** [comment]
Related: zcei/standard-readme#4 (TL;DR: Probably not required by law, but user friendly).

**Comment by RichardLitt** [comment]
> TL;DR: Probably not required by law, but user friendly

Sums it up for me.

**Comment by tunnckoCore** [comment]
> If I just cat the README, I should know the License without having to look in another file.

good point, just forgot about that.

**Comment by RichardLitt** [comment]
Cool. Reopen if more discussion happens; at some point, I've got to stop bikeshedding - doesn't mean I am stopping conversation, though. :)

**Comment by tunnckoCore** [comment]
But one more thing. Why you'll `cat` the README and expect to have license header as last header, when you'll see the LICENSE file when `ls`.

**Comment by RichardLitt** [comment]
Because that's an extra command. And because I think that the README should be a source of truth for this, too. It's often what one finds as the first point of entry - even before downloading the module, like on GitHub, where you have to click License to see the contents of the file.

**Comment by tunnckoCore** [comment]
In 90 or even 99% of the cases there's also license file in every repository, so you can directly `cat` it, not the readme, lol. But nevermind, i'm just thinking differently whole life :D

**Comment by RichardLitt** [comment]
:) No! These are good questions. I just think that the redundancy is super useful here.
